flux-dsl/src/test/java/com/influxdb/query/dsl/functions/restriction/RestrictionsTest.java

```diff
@@ -95,4 +95,14 @@ void emptyLogical() {
         restrictions = Restrictions.or(Restrictions.or());
         Assertions.assertThat(restrictions.toString()).isEqualTo("");
     }
+
+    @Test
+    void escapeStringValues() {
+        Restrictions restrictions = Restrictions.tag("my-tag").equal("escaped\"tag");
+        Assertions.assertThat(restrictions.toString()).isEqualTo("r[\"my-tag\"] == \"escaped\\\"tag\"");
+
+        restrictions = Restrictions.column("my\"column").exists();
+        Assertions.assertThat(restrictions.toString()).isEqualTo("exists r[\"my\\\"column\"]");
+    }
+
```

Comment:
Please remove an empty line:

```suggestion
    @Test
    void escapeStringValues() {
        Restrictions restrictions = Restrictions.tag("my-tag").equal("escaped\"tag");
        Assertions.assertThat(restrictions.toString()).isEqualTo("r[\"my-tag\"] == \"escaped\\\"tag\"");

        restrictions = Restrictions.column("my\"column").exists();
        Assertions.assertThat(restrictions.toString()).isEqualTo("exists r[\"my\\\"column\"]");
    }
```